.github/README.md

```diff
@@ -0,0 +1,33 @@
+## Testing Hypertrace deployment using end to end tests
+
+### Pre-requisites
+- Java 11
+- NPM 
+- Chrome webdriver
+
+### Run GraphQL Tests
+
+To run GraphQL tests, 
+- go to `graphql-e2e-tests` directory using `cd graphql-e2e-tests`. 
+- These GraphQL tests run on specific set of traces so you can ingest these traces using the `ingest-traces.sh` script. 
+- Run `./ingest-traces.sh`
+- Now to run tests, run `gradlew run`. This will execute all the GraphQL tests. 
```

Comment:
Running the test can also be done in the script?

Reply:
It can be. but it's anyways abstracted enough to run with single gradle command so not sure if we should abstract it even more.